restli-server/src/test/java/com/linkedin/restli/internal/server/model/TestRestLiApiBuilder.java

```diff
@@ -198,4 +198,33 @@ public void testActionReturnType(Class<?> resourceClass, Class<?> expectedAction
       Assert.assertEquals(resourceMethodDescriptor.getActionReturnType(), expectedActionReturnType);
     }
   }
+
+  @DataProvider(name = "unstructuredFinderReturnTypeData")
+  private Object[][] unstructuredFinderReturnTypeData()
```

Comment:
I'd recommend avoiding the term "unstructured" here, since it may be confused with another Rest.li feature called "unstructured resources"